modules/systemdstates/charts.go

```diff
@@ -0,0 +1,93 @@
+package systemdstates
+
+import (
+	"github.com/netdata/go-orchestrator/module"
+)
+
+type (
+	// Charts is an alias for module.Charts
+	Charts = module.Charts
+
+	// Dim is an alias for module.Dim
+	Dim = module.Dim
+)
+
+var charts = Charts{
+	{
+		ID:    "service_states",
+		Title: "Systemd Service Units (active => 1, activating => 2, failed => 3, inactive => 4, deactivating => 5)",
```

Comment:
i think we need to do it other way arount: `num => text`, it looks more logical to me because we decode it that way - numeric representation to textual.

And i suggest to use `num: text, ` pattern, it is used it in several collectors

Reply:
(for all charts ofc)